__tests__/app.test.js

```diff
@@ -49,9 +49,27 @@ describe("app", () => {
             topic: "mitch",
             created_at: "2020-06-06T09:10:00.000Z",
             votes: res.body.article.votes,
+            comment_count: 2,
+          });
+        });
+    });
+    test("status: 200 - responds with correct article including comment_count property ", () => {
+      const article_id = 9;
+      return request(app)
+        .get(`/api/articles/${article_id}`)
+        .expect(200)
+        .then((res) => {
+          expect(res.body.article).toEqual({
+            author: "butter_bridge",
+            title: "They're not exactly dogs, are they?",
+            article_id: 9,
+            body: "Well? Think about it.",
+            topic: "mitch",
+            created_at: "2020-06-06T09:10:00.000Z",
+            votes: res.body.article.votes,
+            comment_count: 2,
```

Comment:
I would say the same here - if you add even more properties in future, you'll have 2 tests to go back and change which we'd like to avoid